sormas-app/app/src/main/java/de/symeda/sormas/app/rest/SynchronizeDataAsync.java

```diff
@@ -125,7 +126,7 @@ protected Void doInBackground(Void... params) {
 				}
 				// Pull and remove archived entities when the last time this has been done is more than 24 hours ago
 				if (ConfigProvider.getLastArchivedSyncDate() == null
-					|| DateHelper.getFullDaysBetween(ConfigProvider.getLastArchivedSyncDate(), new Date()) >= 1) {
+					|| DateHelper.getFullDaysBetween(ConfigProvider.getLastArchivedSyncDate(), new Date()) >= 0) {
```

Comment:
Revert this change (probably for testing purposes)

Reply:
changed, sorry for that. I thought I did but did not check before finishing